### PR TITLE
fixed broken link to regex cheat sheet github link

### PR DESCRIPTION
**Pull request by preemeijer**
Hello,
Came across a link that isn't working anymore.
The original github link for the Regex cheat sheet gives a user an 404 message.
Greetings, Paul

### CODE REVIEW

File: lib/DDG/Publisher/Site/Duckduckhack/Root.pm [hunk 1]
```diff
@@ -192,7 +192,7 @@ sub pages {
 			name => 'Regex cheat sheet',
 			desc => 'Quickly see regex syntax',
 			user => 'mintsoft',
-			link => 'https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/lib/DDG/Goodie/RegexCheatSheet.pm',
+			link => 'https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/share/goodie/cheat_sheets/json/regex.json',
 			example_link => 'https://duckduckgo.com/?q=regex+cheat+sheet&t=ffab&ia=answer',
```
Comment (moollaza) [inline lib/DDG/Publisher/Site/Duckduckhack/Root.pm]: we should also change `answer` here to `cheatsheet` so that it opens the IA when the link is visited

**Comment by preemeijer** [comment]
@moollaza: is it ok or do I need to do something more/else?

**Comment by moollaza** [comment]
@preemeijer thanks! 

@jbarrett this LGTM -- should I merge or will it affect your PR where this is being moved to CommPlat?

**Comment by jbarrett** [comment]
I believe all of this is gone now, right? This hash was marked as being used for the old DDH site, so I removed it.

@preemeijer Was this an error you saw on the live site? I may have missed something here

**Comment by preemeijer** [comment]
@moollaza and @jbarrett: I see that this pull request has conflicts right now.
The link under Staff Pick Instant Answers to the regex cheat sheet is broken. So I thought that replacing the correct link to the github page is the solution. :)
If you now click on the link you get a 404 error page from Github.

**Comment by jbarrett** [comment]
Many thanks @preemeijer! 

This fix is spot on - turns out I broke things pretty badly in another task then confused myself. This should be merged soon enough.

**Pull request merged**